utils/retryexecutor_test.go

```diff
@@ -13,7 +14,7 @@ func TestRetryExecutorSuccess(t *testing.T) {
 
 	executor := RetryExecutor{
 		MaxRetries:      retriesToPerform,
-		RetriesInterval: 0,
+		RetriesInterval: 5000,
```

Comment:
Is this change actually necessary? Wouldn't it make the test no longer, with no additional testing value?